api/src/pcapi/routes/backoffice/templates/admin/subcategories.html

```diff
@@ -0,0 +1,62 @@
+{% from "components/forms.html" import build_form_fields_group with context %}
+{% extends "layouts/connected.html" %}
+{% block page %}
+  <div class="pt-3 px-5 pc-feature-flip">
+    <h2 class="fw-light">Liste des sous-catégories</h2>
+    <div class="row">
+      <div class="col-12">
+        <table class="table mb-4"
+               data-table-multi-select-id="table-subcategories">
+          <thead>
+            <th scope="col">pro_label</th>
+            <th scope="col">app_label</th>
+            <th scope="col">Nom tech de la sous-catégorie</th>
+            <th scope="col">Nom tech de la catégorie</th>
+            <th scope="col">native_category</th>
+            <th scope="col">search_group_name</th>
+            <th scope="col">homepage_label_name</th>
+            <th scope="col">is_event</th>
+            <th scope="col">conditional_fields</th>
+            <th scope="col">can_expire</th>
+            <th scope="col">is_physical_deposit</th>
+            <th scope="col">is_digital_deposit</th>
+            <th scope="col">online_offline_platform</th>
+            <th scope="col">reimbursement_rule</th>
+            <th scope="col">can_be_duo</th>
+            <th scope="col">can_be_educational</th>
+            <th scope="col">is_selectable</th>
+            <th scope="col">Réservable par les 15-17 si gratuite</th>
+            <th scope="col">Réservable par les 15-17 si payante</th>
+            <th scope="col">can_be_withdrawable</th>
+          </thead>
+          <tbody>
+            {% for subcategory in rows %}
+              <tr>
+                <td>{{ subcategory.pro_label }}</td>
+                <td>{{ subcategory.app_label }}</td>
+                <td>{{ subcategory.id }}</td>
+                <td>{{ subcategory.category.id }}</td>
+                <td>{{ subcategory.native_category }}</td>
```

Comment:
On doit raisonnablement pouvoir se passer de `NativeCategory.` à l'affichage : 
```suggestion
                <td>{{ subcategory.native_category.name }}</td>
```